nginx-directory/configfile.libsonnet

```diff
@@ -33,13 +33,14 @@ function(services) {
     + (
       if service.subfilter
       then |||
+        proxy_set_header Accept-Encoding "";
         sub_filter 'href="/' 'href="/%(path)s/';
         sub_filter 'src="/' 'src="/%(path)s/';
         sub_filter 'action="/' 'action="/%(path)s/';
         sub_filter 'endpoint:"/' 'endpoint:"/%(path)s/';  # for XHRs.
         sub_filter 'href:"/v1/' 'href:"/%(path)s/v1/';
         sub_filter_once off;
-        sub_filter_types %(rendered_subfilter_content_types)s;
+        sub_filter_types %(rendered_subfilter_content_types)s';
```

Comment:
Is this correct? Shouldn't have a single quote also at the beginning of the "value"?

Reply:
This change shouldn't be there, going to revert it.

Reply:
Removed it in https://github.com/grafana/jsonnet-libs/pull/793/commits/a4f1cd87c7e2c3bebe1aaf761edefc5c1d5ed09c